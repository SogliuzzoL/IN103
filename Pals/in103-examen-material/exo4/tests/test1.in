1
6
7
8
4
3
9
10
2
5
