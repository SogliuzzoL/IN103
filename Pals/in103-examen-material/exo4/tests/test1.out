-1
12
48
62
60
59
75
93
92
92
